packages/dds/tree/src/test/simple-tree/toMapTree.spec.ts

```diff
@@ -1294,6 +1294,8 @@ describe("toMapTree", () => {
 				policy: {
 					fieldKinds,
 					validateSchema: true,
+					// TODO: Unit test leveraging this policy in this file
```

Comment:
it's not the main purpose of this PR, so I left these out, but would be happy to write some now or do a quick follow-up PR.

Reply:
not relevant in latest